vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -17,7 +17,7 @@ def save_vms_inventory(ems, hashes, target = nil)
     log_header = "MIQ(#{self.name}.save_vms_inventory) EMS: [#{ems.name}], id: [#{ems.id}]"
 
     disconnects = if target.kind_of?(ExtManagementSystem) || target.kind_of?(Host)
-      target.vms_and_templates(true).dup
+      target.vms_and_templates(true).to_a.dup
```

Comment:
wonderful find.

there are a bunch of `parent.child(true).dup` calls for assigning a variable called `disconnects` and `deletes`.

@tenderlove Are you saying that we won't want to blindly change all of them? Any special use case to keep in mind?